mythril/laser/smt/__init__.py

```diff
@@ -56,6 +66,17 @@ class _SmtSymbolFactory(SymbolFactory):
     the classes in: mythril.laser.smt
     """
 
+    @staticmethod
+    def Bool(value: bool, annotations=None):
+        """
+        Creates a Bool with concrete value
+        :param value: The boolean value
+        :param annotations: The annotations to initialize the bool with
+        :return: The freshly created Bool()
+        """
+        raw = z3.Bool(value)
+        return Bool(raw)
```

Comment:
the annotations need to be passed as a parameter right?

Reply:
argh!